logger/logger.go

```diff
@@ -123,9 +124,24 @@ func Err(err error) *zerolog.Event {
 	return Logger.Err(err)
 }
 
+// Ctx returns a DBSQLLogger from the provided context. If no logger is found,
+// the default logger is returned.
+func Ctx(ctx context.Context) *DBSQLLogger {
```

Comment:
If find this function name a bit confusing.  I know that naming things here is complicated by the use of the noun context and the type context but it would be good to have a function name that more clearly tells you what it is doing.  Maybe something like FromCtx?  I'm open to suggestions.